modules/base/rendering/renderablesphere.cpp

```diff
@@ -386,14 +354,14 @@ void RenderableSphere::render(const RenderData& data, RendererTasks&) {
         glDisable(GL_CULL_FACE);
     }
 
-    if (_useAdditiveBlending) {
+    if (renderBin() == Renderable::RenderBin::PreDeferredTransparent) {
```

Comment:
More readable if accessing the `_renderBin` directly rather than going through the getting mething